### ISSUE TITLE
Sync failed error

### ISSUE THREAD

**Issue by Gamecreator2002** [GedasFX/decky-cloud-save#103]
I recently factory reset my steam deck and had previously used this plugin with no issues at all, but after the reset I can't seem to get this to work. I'm very new to linux and running command codes so when I look at these logs I see nothing wrong but still I continue to get the same error message when I go back into game mode. 
![20240908_132119 1](https://github.com/user-attachments/assets/61ea70a7-40df-44a8-8303-b6cd4c0ce93a)
![Screenshot 2024-09-10 002636](https://github.com/user-attachments/assets/689e1c0a-3551-4d2b-9a6f-bc4119a86e73)

**Comment by GedasFX** [comment]
yeah you see the top of the log, you should scroll a bit down. Also try out the testing store version, hoping to merge that to main store soon enough.

**Comment by Gamecreator2002** [comment]
Ahh I see! I had to increase the window size to see the error. 
I'm getting a **"failed to create file system for 'backend:decky-cloud-save': unable to get drive_id and drive_type - if you are upgrading from older versions of rclone, please run 'rclone config' and re-configure this backend"** 
followed by another error **"failed to query root for drive " 'b!13ZndwGDf0qyiUYpV1SmS2WaMboU5udBre-wBKiIyRa7uj740zqEQJWSHhAJ4A7n': HTTP error 400 (400 Bad Request) returned body: '{\'error\':.....**
I don't have any other version of rclone or anything, and when attempting to put thr rclone config command into the Konsole nothing came up. What is this error telling me? (Also please ignore me closing and reopening this issue that was a github noob moment I had)
![17260868094781380020899589223473](https://github.com/user-attachments/assets/c32e23e8-13c5-414e-aa28-be7db2a83fdc)
![17260868641513662890335349391608](https://github.com/user-attachments/assets/901b0b2f-59aa-4742-8120-ace2d76a76d1)
![17260869005145376702268328824763](https://github.com/user-attachments/assets/474655b6-2cce-4f2f-b27c-f619479bced9)

**Comment by GedasFX** [comment]
Actually looks like it was merged 🎉 
https://github.com/SteamDeckHomebrew/decky-plugin-database/pull/637
Try with latest version.

**Comment by Gamecreator2002** [comment]
Ok yeah I just updated and checked out #99 , I'm using OneDrive and for some reason it doesn't want to cooperate. Here's the logs I'm getting now + a screenshot of what my screen looks like after signing into Microsoft:
![17262398793598431131870211468435](https://github.com/user-attachments/assets/283e2cf7-5d93-4546-92b0-451d9ca4f16f)
![17262399343151785826131454948657](https://github.com/user-attachments/assets/465f40b6-4aa4-4da4-845b-ea66dd0ce56f)

How do I get to the testing store to try that one out?

**Comment by GedasFX** [comment]
It should be on live store now, v1.4.2.

It really does not like OneDrive now huh... Blank screen can be caused by other reasons, try fully rebooting steam deck,

**Comment by Gamecreator2002** [comment]
Full reboot as in another factory reset?

**Comment by GedasFX** [comment]
nah just restart

**Comment by Gamecreator2002** [comment]
Unfortunately restarting did nothing but on the other hand I did find the rclone config. Is there anything I could do in there?

**Comment by AT181903** [comment]
Same problem here

**Comment by ZATYGOLD** [comment]
I am having the same issue. My Decky Cloud Saves states "Currently using: OneDrive" but I still get the "Failed to create file system" error. Unable to get drive_id and drive_type. I will follow the suggested instructions to run `rclone config` and reconfigure it.

**Comment by oOtroyOo** [comment]
Same error , latest on decky store

btw, use desktop mode, run  quickstart/onedrive.sh or drive.sh, could launch a browser whatever you installed. to fix black screen, or google trust browser

**Comment by dimateos** [comment]
I got the same errors. Seems duplicate of https://github.com/GedasFX/decky-cloud-save/issues/99

Also black screen, so had to get the OneDrive token in desktop mode. Fine for me tho.

The rclone config created seems to be outdated as hinted by `if you are upgrading from older versions of rclone, please run 'rclone config' and re-configure this backend`. 
* The config file is missing `drive_id` (ending ID of your onedrive URL `&cid=XXXXXXXXXXXX`) and `drive_type` (personal).
* You can also run `rclone config` and follow the steps, but a bit harder as the config file is in the homebrew/settings folder (I moved it next to rclone and back)
* After this it worked fine for me!

Maybe updating rclone as suggested in https://github.com/GedasFX/decky-cloud-save/issues/107 might fix this?

I saw you comment in https://github.com/GedasFX/decky-cloud-save/issues/120#issuecomment-2739682380, but maybe this is an easy fix if the update is enough.

**Comment by TuanWorkMail** [comment]
I am using SDH-GameSync and running into the same error, manually running `rclone config` and select `3` at `drive_id` worked.
